.github/workflows/msys2-htslib-build.yml

```diff
@@ -12,7 +12,7 @@ on:
       - 'scripts/**'
   workflow_dispatch:
 env:
-  PACKAGE_VERSION: "1.20"
+  PACKAGE_VERSION: "1.21"
```

Comment:
Note that [1.21-0](https://github.com/TileDB-Inc/m2w64-htslib-build/releases/tag/1.21-0) was already built and uploaded since this PR uses an internal branch

https://github.com/TileDB-Inc/m2w64-htslib-build/blob/562caac9ca4d29250d032bd2b10b37ad5abaf168/.github/workflows/msys2-htslib-build.yml#L59

Reply:
Yeah I saw it. I will delete the artifact before merging.

Reply:
Done, binaries published.